com.logseq.Logseq.desktop

```diff
@@ -1,9 +1,16 @@
 [Desktop Entry]
-Type=Application
-NoDisplay=false
+Name=Logseq Desktop
+Exec=run.sh %U --enable-features=UseOzonePlatform --ozone-platform-hint=auto --enable-features=WaylandWindowDecorations
```

Comment:
I'll need to test this. There's a Wayland branch on this repository which uses similar options. IIRC the auto hint will cause crashes if the desktop is not Wayland. Maybe it's already fixed by the latest electron?